packages/angular_devkit/build_angular/src/ng-packagr/index.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { BuilderContext, BuilderOutput, createBuilder } from '@angular-devkit/architect';
+import { resolve } from 'path';
+import { Observable, from } from 'rxjs';
+import { mapTo, switchMap } from 'rxjs/operators';
+import { Schema as NgPackagrBuilderOptions } from './schema';
+
+async function initialize(
+  options: NgPackagrBuilderOptions,
+  root: string,
+  // tslint:disable-next-line:no-implicit-dependencies
```

Comment:
With the peer dependency in place, these tslint comments shouldn't be needed

Reply:
You are correct, updated!